src/App.tsx

```diff
@@ -1,9 +1,21 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const filterMovies = () => {
+    const filter = query.toLowerCase();
+
+    return moviesFromServer.filter(movie => (
+      movie.title.toLowerCase().includes(filter)
+      || movie.description.toLowerCase().includes(filter)));
```

Comment:
you basically do the same thing with both strings here, create a helper function for it

Reply:
trim method crashes tests